src/server/components/lists/router.ts

```diff
@@ -2,16 +2,16 @@
 import express from "express";
 import * as Controllers from "./controllers";
 import { listsRoutes } from "./routes";
-import { csrfRequestHandler } from "server/components/cookies/helpers";
 import { ingestRouter } from "server/components/lists/controllers/ingest/router";
 import annualReviewRouter from "server/components/annual-review/router";
+import { findRouter } from "./find/router";
+import { redirectToFindResource } from "server/components/lists/find/helpers/redirectToFindResource";
+import { loadQueryParametersIntoSession } from "server/components/lists/find/middleware/loadQueryParametersIntoSession";
 
 export const listsRouter = express.Router();
 
-listsRouter.get(listsRoutes.finder, csrfRequestHandler, Controllers.listsGetController);
-listsRouter.post(listsRoutes.finder, csrfRequestHandler, Controllers.listsPostController);
-listsRouter.get(listsRoutes.removeLanguage, csrfRequestHandler, Controllers.removeLanguageGetController);
```

Comment:
Not sure why it was done this way..
`/language/:language/remove` doesn't follow any patterns. It doesn't make sense to `/language/:language/remove` to delete something on the `/find/*` resource